pkg/store/driver/txn/unionstore_driver.go

```diff
@@ -157,6 +157,16 @@ func (m *memBuffer) MayFlush() error {
 	return err
 }
 
+// GetLocal implements kv.MemBuffer interface
+func (m *memBuffer) GetLocal(ctx context.Context, key []byte) ([]byte, error) {
+	data, err := m.MemBuffer.GetLocal(ctx, key)
+	return data, derr.ToTiDBErr(err)
+}
+
+func (m *memBuffer) BatchGet(ctx context.Context, keys [][]byte) (map[string][]byte, error) {
+	return m.MemBuffer.BatchGet(ctx, keys)
```

Comment:
Does it need `derr.ToTiDBErr(err)`? Everything is an error. It's so obscure.

Reply:
Yes, added.